akka-samples/akka-docs-java-lambda/pom.xml

```diff
@@ -6,7 +6,7 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <publishedAkkaVersion>2.3.7</publishedAkkaVersion>
+    <publishedAkkaVersion>2.3.9</publishedAkkaVersion>
```

Comment:
this is the stream dependency, and that must be fixed (2.3-SNAPSHOT would be from the release-2.3-dev branch)
messy, but things will be cleaned up when we leave this -dev branch